GVFS/GVFS.UnitTests/Common/LibGit2RepoInvokerTests.cs

```diff
@@ -25,32 +23,65 @@ public void Setup()
             this.invoker?.Dispose();
 
             this.tracer = new MockTracer();
-            this.invoker = new LibGit2RepoInvoker(this.tracer, this.CreateRepo, this.disposalPeriod);
             this.numConstructors = 0;
             this.numDisposals = 0;
             this.DisposalTriggers = new BlockingCollection<object>();
+
+            this.invoker = new LibGit2RepoInvoker(this.tracer, this.CreateRepo);
         }
 
         [TestCase]
-        public void DoesNotCreateRepoOnConstruction()
+        public void DoesCreateRepoOnConstruction()
         {
-            this.numConstructors.ShouldEqual(0);
+            this.numConstructors.ShouldEqual(1);
         }
 
         [TestCase]
-        public void CreatesRepoOnTryInvoke()
+        public void CreatesOnRequestAfterClosed()

```

Comment:
I'm not sure my understanding of what this test is validating is correct.  Is it that `InitializedSharedRepo` creates a LibGit2 after `DisposeSharedRepo` has been called?

Reply:
`Initialize[d]SharedRepo` had an extra 'd' which was a typo and may have added to the confusion.

I will rename this test `CreatedByInitializeAfterClosed`.